corehq/apps/sms/change_publishers.py

```diff
@@ -23,9 +23,9 @@ def change_meta_from_sms(sms):
     from corehq.apps.change_feed import data_sources
     return ChangeMeta(
         document_id=sms.couch_id,
-        data_source_type=data_sources.SMS,
+        data_source_type=data_sources.SOURCE_SQL,
         data_source_name=data_sources.SMS,
-        document_type='SMS',
+        document_type=topics.SMS,
```

Comment:
no, since SMS doesn't have a `doc_type` we just use the topic. Also mentioned here: https://github.com/dimagi/commcare-hq/blob/ba2043bf9b6f9ff83b74164bbbb321bc9ae23a0f/corehq/apps/change_feed/topics.py#L76